src/main/java/com/cs56fitnessapp/models/User.java

```diff
@@ -19,4 +19,17 @@
     private Goal goal;
     private double goalWeight;
     private ActivityLevel activityLevel;
+
+    public double  getBMI(){
+
+        return 0;
+    }
+
+    public double getbodyFatPercentage(){
+
+
+        return 0;
+    }
+
```

Comment:
Currently your methods `getBMI()` and `getbodyFatPercentage()` are not implemented. They always return 0 no matter what user's attributes are. E.g If I create a new user with height 177cm and body mass 58kg - her BMI would still be 0 according to `getBMI()`. 

These methods should return the actual values. 
They must use `FitnessFormulas.BMI()` and `FitnessFormulas. bodyFatPercentage()` to do that.